### PR TITLE
Cleaned up landscape config wizard

### PR DESCRIPTION
**Pull request by silverdrake11**

### CODE REVIEW

File: landscape/client/configuration.py [hunk 1]
```diff
@@ -805,7 +819,7 @@ def report_registration_outcome(what_happened, print=print):
     human-readable form.
     """
     messages = {
-        "success": "System successfully registered.",
+        "success": "Registration request sent successfully.",
```
Comment (Perfect5th) [inline landscape/client/configuration.py:822]: Oh thank you for adjusting this to be more accurate.

**Review by Perfect5th — COMMENTED** [review]
1 nit comment and one more important one.

File: landscape/client/configuration.py [hunk 2]
```diff
@@ -473,45 +443,97 @@ def query_tags(self):
...
+            if self.config.account_name == "standalone":
+                self.config.account_name = ""
+
+    def show_summary(self):
+
+        tx = f"""A summary of the provided information:
+            Computer's Title: {self.config.computer_title}
+            Account Name: {self.config.account_name}
+            Landscape FQDN: {self.get_domain()}
+            Registration Key: {True if self.config.registration_key else False}
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply landscape/client/configuration.py:489]: It's not being displayed. It's True or False

File: landscape/client/configuration.py (hunk 1 above)
Comment (silverdrake11) [reply landscape/client/configuration.py:822]: You can thank Rajan

File: landscape/client/configuration.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Perfect5th) [reply landscape/client/configuration.py:489]: :facepalm:

**Review by Perfect5th — APPROVED** [review]

File: landscape/client/configuration.py [hunk 3]
```diff
@@ -473,45 +443,97 @@ def query_tags(self):
...
-                )
-                self.config.tags = None  # Reset for the next prompt
-            else:
-                break
-
-    def show_header(self):
-        show_help(
+        options = self.config.get_command_line_options()
+        if "ping_url" in options:
+            if "url" in options:
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply landscape/client/configuration.py]: Fixed

**Review by Perfect5th — APPROVED** [review]

**Pull request merged**